db/single/case_sensitive/image_list.txt

```diff
@@ -0,0 +1,47 @@
+425/abandoned-2179173_1280.jpg 425
```

Comment:
I guess you need to adjust this one too.

Reply:
done